lib/resolveurl/plugins/wstream.py

```diff
@@ -26,7 +26,7 @@ class WstreamResolver(ResolveGeneric):
 
     def get_media_url(self, host, media_id):
         return helpers.get_media_url(self.get_url(host, media_id),
-                                     patterns=[r'''sources:\s*\[{file:\s*"(?P<url>[^"]+)'''],
+                                     patterns=[r'''sources:\[\"(?P<url>[^\"]+)\"\]'''],
```

Comment:
```
patterns=[r'''sources:\s*\[{file:\s*"(?P<url>[^"]+)''',
          r'''sources:\s* \["(?P<url>[^"]+)'''],
```
would be better

Reply:
But with my change wstream work with r'''sources:\s* \["(?P<url>[^"]+)'''], not work

Reply:
But whit this patterns patterns=[r'''sources:\s*\[{file:\s*"(?P<url>[^"]+) wstream not work

Reply:
both patterns are there in the change and work correctly now. Check commit https://github.com/jsergio123/script.module.resolveurl/commit/0eebd297571f95ecb2d01f78e91e3e0f38eb2520